ui/pages/build-status.js

```diff
@@ -88,6 +88,16 @@ const RepoStatusItem = ({ repo, organization = 'Mender', branch = 'master', cove
   </Stack>
 );
 
+const buildStatusColor = status => {
+  const buildStatusColorMap = {
+    'FAILED': 'error',
+    'RUNNING': 'warning',
+    'SUCCESS': 'success',
+    default: 'warning.dark' // WTF is going on colour!
+  };
+  return buildStatusColorMap[status] || buildStatusColorMap.default;
+};
```

Comment:
```suggestion
const buildStatusColorMap = {
    FAILED: 'error',
    RUNNING: 'warning',
    SUCCESS: 'success',
    default: 'warning.dark' // WTF is going on colour!
};
```
mind the lack of `'` for the uppercase entries... consistency! (unless of course we enter `'09-FAILED-ONLY-PARTLY-%': 'grey'` into the map)